client/src/components/BlockEditor/BlockEditorEditing.tsx

```diff
@@ -201,7 +210,29 @@ export class BlockEditorEditingPresentational extends React.Component<
     }
   };
 
-  private onChangeCallback = (c: any) => {
+  private onChangeCallback = (c: Change) => {
+    // first check to see if document changed, which we can do with !== b/c
+    // Slate uses immutable objects, if it has changed then normalize wrt
+    // pointer spacing
+
+    if (c.value.document !== this.props.block.value.document) {
+      c = normalizeExportSpacing(c);
+    }
+
+    // the rest of this method before the last two lines are trying to prevent
+    // the user from using a click top get the text cursor in off limit spacing
+    // the onClick handler didn't seem to have access to the new cursor position
+    // so this logic lives here instead
+
+    const value = c.value;
+    const selectionIsExpanded = value.selection.isExpanded;
+
+    // if selection is expanded, then we don't do this
+    // this allows mouse dragging to select across pointers
+    if (!selectionIsExpanded && isCursorInPotentiallyProblematicPosition(value)) {
```

Comment:
Can these two checks be in the file for ``handleStationaryCursor``, given that it already has other checks?